doc/timeToString.md

```diff
@@ -0,0 +1,25 @@
+# ASAB WebUI timeToString function
+
+`timeToString` is a function returning date and time as a simple string in a format based on passed `lang` parameter. 
+Acccepts 2 paramaters
+    - timestamp - in seconds or miliseconds
+    - language object (coming from `useDateFNSLocale()`) of selected locale - default `undefined` shows `en` format (e.g. `'Apr 11, 2022, 1:16 PM'`)
+
+## Example code
+
+```javascript
+import { timeToString, useDateFNSLocale } from 'asab-webui';
+
+const MyComponent = () => {
+    const lang = useDateFNSLocale();
+
+    return (
+        <>
+            Current date and time is {timeToString(new Date(), lang)}
```

Comment:
@petrKavulok Well, for this case, I may use the original component, since its in the render part. Can you please add the documentation, how to use timeToString also outside of render (e.g. as a value return)?

Or do it different way - create one doc about DateTime component, and inside of it, add sections with examples how to use the original `DateTime` component in render, and how to use timeToString method.